### PR TITLE
[BIOMAGE-1919] Remove account specific hardcoding from the UI

### PR DESCRIPTION
**Pull request by ivababukova**
# Description
This change, combined with the changes in the linked iac PR remove the need to provide account id for the iam role via Github Actions. The iam role is used by the UI pod and is inserted as the `AWS_ROLE_ARN` environment variable.

# Details
#### URL to issue
https://biomage.atlassian.net/browse/BIOMAGE-1919

#### Link to staging deployment URL (or set N/A)
N/A

#### Links to any PRs or resources related to this PR
https://github.com/hms-dbmi-cellenics/iac/pull/387/files

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -289,15 +289,14 @@ jobs:
           | yq w -d1 - metadata.labels.sandboxId "$SANDBOX_ID" \
           | yq w -d1 - "metadata.annotations[filter.fluxcd.io/chart-image]" "$IMAGE_GLOB" \
           | yq w -d1 - spec.releaseName "$DEPLOYMENT_NAME" \
-          | yq w -d1 - spec.chart.ref "$CHART_REF" \
```
Comment (aerlaut) [inline .github/workflows/ci.yaml]: Why did you delete this?

Comment (ivababukova) [reply .github/workflows/ci.yaml]: this is still needed, so I have added it back

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -297,7 +297,7 @@ jobs:
           | yq w -d1 - spec.values.image.repository "$DEPLOYMENT_NAME" \
           | yq w -d1 - spec.values.image.tag "$IMAGE_TAG" \
           | yq w -d1 - spec.values.replicaCount $REPLICA_COUNT \
-          | yq w -d1 - spec.values.serviceAccount.iamRole "arn:aws:iam::${{ steps.setup-aws.outputs.aws-account-id }}:role/ui-role-$KUBERNETES_ENV" \
```
Comment (StefanBabukov) [inline .github/workflows/ci.yaml:300]: so the accound id is not needed for the name of the iam role right? just ui-role-env

**Review by StefanBabukov — APPROVED** [review]
